### ISSUE TITLE
Capricious Cassie's Respawn Time Incorrect

### ISSUE THREAD

**Issue by Aneyus** [EdenServer/community#1480]
Seems to possibly be using the current faster respawn on retail? Current wiki entry says:
Timed spawn around (G-6) underground.
Exact respawn time is currently unknown; it is less than 12 hours.

Old wiki entry says:
Respawn around (G-6) underground approximately every 21-24 hours.
https://ffxiclopedia.fandom.com/wiki/Capricious_Cassie?oldid=242167

**Comment by setsuk0** [comment]
Can confirm I've had issues with this one this week. I killed it one day and when 21-24hr window was up it did not spawn. It did, however, spawn a few hours later putting it at almost a 30 hour (unless it had been killed somewhere in the middle by someone else). When its 21-24 hr window was back up yesterday, it did not spawn, and instead I tracked a ToD that would have been 36 hours from my ToD unless it spawned again and was killed somewhere in the middle with the 12 hour time frame stated on current retail.
